disspy/bot.py

```diff
@@ -602,6 +533,18 @@ def run(
         :param status: Status for bot user
         :return: None
         """
+        # Type checks
+        class _StatusTypeCheck:
```

Comment:
May be create CreateTypeCheckClass(__values__, __type__) method in utils.py returns a class with __values__ and __type__ methods?